man/docker-run.1.md

```diff
@@ -252,6 +253,11 @@ redirection on the host system.
                                'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
+**--isolation**="*default*"
+   Isolation specifies the type of isolation technology used by containers. On Linux, the only supported option is 'default' (Linux Namespaces). Windows supports 'default' (the daemon default specified by `--exec-opt isolation=`), 'process' (namespace isolation only), or 'hyperv' (Hyper-V hypervisor partition based isolation).
```

Comment:
Rather than trying to cover all this in the  flag, please keep the command simple and add the content you created for `run` above.